### PR TITLE
PE 1178/Folder Uploads

### PR DESCRIPTION
**Pull request by jdaev**
Remove the 2 data item limit for bundle creation and instead take the number of bundles from the handle.

### CODE REVIEW

File: lib/src/models/data_bundle.dart [hunk 1]
```diff
@@ -40,35 +41,61 @@ class DataBundle {
...
+            // Add header to array of headers
+            headers.setAll(64 * dataItemIndex, header);
+            dataItemIndex++;
+            // Convert to array for flattening
+            binaries.add(raw.takeBytes());
+          }
+        },
+      ),
+    );
+    await Future.wait(
```
Comment (thiagocarvalhodev) [inline lib/src/models/data_bundle.dart]: The logic here is the same of the above. What do you think about split it on another function?

Comment (jdaev) [reply lib/src/models/data_bundle.dart]: Looks like you're reviewing the changes from the first commit. This was updated.

Comment (thiagocarvalhodev) [reply lib/src/models/data_bundle.dart]: Yes! Sorry, Javed.

**Review by thiagocarvalhodev — DISMISSED** [review]
LGTM!

**Review by karlprieb — DISMISSED** [review]

File: lib/src/utils/implementations/bundle_tag_parser_js.dart [hunk 2]
```diff
@@ -15,12 +15,12 @@ external List<BundleTag> deserializeTagsFromBuffer(var buffer);
 
 Uint8List serializeTags({required List<Tag> tags}) {
   final decodedTags = <Tag>[];
-  tags.forEach((tag) {
+  for (var tag in tags) {
```
Comment (karlprieb) [inline lib/src/utils/implementations/bundle_tag_parser_js.dart:18]: why not `final`?

**Review by thiagocarvalhodev — DISMISSED** [review]

**Review by fedellen — APPROVED** [review]

**Pull request merged**